drivers/sx127x/sx127x_netdev.c

```diff
@@ -490,7 +484,6 @@ static int _set_state(sx127x_t *dev, netopt_state_t state)
 
         case NETOPT_STATE_IDLE:
             /* set permanent listening */
-            sx127x_set_rx_timeout(dev, 0);
```

Comment:
I think this is breaking the case where the LoRa radio is kept in listen state (this is how I understand `NETOPT_STATE_IDLE`).

Reply:
This `sx127x_set_rx_timeout` only sets a timeout timer. Since now with this PR they are fixed, we don't need to set this value